src/Voron/Data/Tables/Table.cs

```diff
@@ -97,7 +97,7 @@ private void OnDataMoved(long previousId, long newId, byte* data, int size)
         /// Using this constructor WILL NOT register the Table for commit in
         /// the Transaction, and hence changes WILL NOT be commited.
         /// </summary>
-        public Table(TableSchema schema, string name, Transaction tx, int tag)
+        public Table(TableSchema schema, string name, Transaction tx, int tag, bool skipSchemaVerification = true)
```

Comment:
Should be better to use "doSchemaValidation" instead of "skipSchemaValidation" and invert the logic.